README.md

```diff
@@ -173,8 +173,7 @@ Here are the common options:
             anchor: new google.maps.Point(0, 32)
         },
         label: function(point) {
-            // a function that return the text value of the label
-            return point.data.location.name;
+            return '<div class="map-custom-label">' + point.id + '</div>';
```

Comment:
``<div class="map-custom-label">${point.id}</div>`` :p ?

Reply:
This expression was as is in previous versions of README.md. I choosed the 'no surprise' strategy to show that there is no breaking change here. But I totally aggree that a template literal would be better here.